queue/queue.py

```diff
@@ -0,0 +1,51 @@
+class Queue(object):
+
+    def __init__(self):
+        self._list = []
+
+    def count(self):
+        return len(self._list)
+
+    def is_empty(self):
+        if self.count() == 0:
+            return True
+        else:
+            return False
+
+    def enqueue(self, item):
+        self._list.append(item)
+
+    def dequeue(self):
+        try:
+            return self._list.pop(0)
+        except IndexError:
+            raise IndexError('pop from empty stack')
+
+
+def main():
+    queue = Queue()
+    n = 100
+
+    print('Empty queue: {0}'.format(queue.is_empty()))
+
+    while queue.count() < 5:
+        print('pushing elements: {0}'.format(n))
+        queue.enqueue(n)
+        n = n + 100
+
+    print('Number of items: {0}'.format(queue.count()))
+    print('Empty queue: {0}'.format(queue.is_empty()))
+
+    while queue.count() >= -20:
```

Comment:
It seems to be a little confusing to put size as -20. You can use `while not queue.is_empty()`, or if you intend to show the exception part, you can use `while True`.

Reply:
yes, in fact I wanted to show the exception part.
Thank you, I'll use ```while True``` instead